src/core/thread/mle_types.hpp

```diff
@@ -258,9 +258,13 @@ enum
 {
     kRegistrationDelayDefault         = 1200, //< In seconds.
     kMlrTimeoutDefault                = 3600, //< In seconds.
+    kMlrTimeoutMin                    = 300,  //< In seconds.
     kBackboneRouterRegistrationJitter = 5,    //< In seconds.
 };
 
+static_assert(kMlrTimeoutDefault >= kMlrTimeoutMin,
```

Comment:
Generally user should not change this internal header file which defines  `kMlrTimeoutDefault` and `kMlrTimeoutMin`,  is it necessary for this `static_assert`?

Reply:
I agree we will hardly change these values.

Reply:
I tends to keep this `static_assert` because it satisfies logical completeness to guarantee that `MLR timeout >= 300` is always true for OT devices.